src/treemap.ts

```diff
@@ -128,6 +135,8 @@ export class TreeMap {
     if (node.dom) return node.dom;
     const dom = document.createElement('div');
     dom.className = NODE_CSS_CLASS;
+    this.options.classList.forEach(nodeClass => dom.classList.add(nodeClass(node)));
```

Comment:
This API doesn't make sense to me.  Why accept an array of functions, rather than just one?  What's the use case?

Maybe a single function that can return an array makes more sense?
Maybe we accept an option that lets us just pass each node to the caller to let them mutate as they will, modifying classList or anything else?

Reply:
I like the idea of having a `mutations` option, it could solve a variety of use cases. Great call, will replace this.